Malaba6/Auto-Mart-Fe
# #173020482 - User should Register and li

## What does this PR do?
-  Adds the Signup page
-  Adds the login page

## Description of Task to be completed?
- Create a compoenent to handle form events
- Create signup Component for sign up page 
- Create login component for the login page
- Add Redux actions, Store and Reducers to handle Signup and Login Pages

## How should this be manually tested?
- clone the repository,then run `npm i` to install all depandencies.
- start the server with `npm start`
- Go to `http://localhost:8080/signup` to see the signup page
- Go to `http://localhost:8080/login` to see the loginpage 

## What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/173020482


## Did you test this on all Browsers
